examples/gcp-lakehouse/modules/gcp-sa-provisionning/main.tf

```diff
@@ -0,0 +1,149 @@
+resource "google_service_account" "sa2" {
+  account_id   = "${var.prefix}-sa2"
+  display_name = "Service Account for Databricks Provisioning"
+}
+
+output "service_account" {
```

Comment:
it makes sense to put all outputs into a separate file, like [here](https://github.com/databricks/terraform-databricks-examples/blob/main/modules/aws-workspace-basic/outputs.tf), so registry will show them correctly: https://registry.terraform.io/modules/databricks/examples/databricks/latest/submodules/aws-workspace-basic

Reply:
makes sense. done. Should I break it down apart more to match the granularity of the aws workspace one ?